Management/DomainModels/Location.cs

```diff
@@ -7,26 +8,14 @@ namespace Management.DomainModels
     /// </summary>
     public class Location
```

Comment:
Mengwen, bite me :P

I changed a lot for countrycode and state part. Please read the change first and welcome to discuss with me directly!